backtrace-library/src/main/java/backtraceio/library/models/BacktraceData.java

```diff
@@ -26,6 +26,13 @@
 
     private static transient String LOG_TAG = BacktraceData.class.getSimpleName();
 
+    /**
+     * 16 bytes of randomness in human readable UUID format
```

Comment:
Why we add here a comment that this is a 16 bytes of randomness UUID if it's progruard?